test/formulas/inverse_cases.hpp

```diff
@@ -598,6 +598,28 @@ expected_results expected[] =
         {00000000.00000000000000000000, 0.00000000000000000000, 0.00000000000000000000, 000000.00000000000000000000, 1.00000000000000000000},
         {20020712.84987257421016693115, 180.00000000000000000000, 0.00000000000000000000, 33590.79639541531651047990, -1.00525773151080621837}
     }*/
+    // pi  = 3.1415926535897931
+    // r2d = 57.295779513082323
+    //,
+    /*
+    {
+        {0.26387320519564444 * 57.295779513082323, 0.84008767903244352 * 57.295779513082323},
+        {0.26387320679785603 * 57.295779513082323, 0.84008767099694759 * 57.295779513082323},
+        // TODO: Check Karney
+        {0.051647209427812361, 172.39750897806516, 172.39750904523200, 0.051638407375539180, 0.99999999999999734, true},
+        {0.051647209427812361, 172.39750897806516, 172.39750904523200, 0.051638407375539180, 0.99999999999999734},
+        {0.00000000000000000, 0.00000000000000000, 0.00000000000000000, 0.00000000000000000, 1.0000000000000000},
+        {0.00000000000000000, 0.00000000000000000, 0.00000000000000000, -0.051193490763866173, 1.0000000000000000}
+    }, {
+        {0.26387320519564444 * 57.295779513082323, 0.84008767903244352 * 57.295779513082323},
+        {(0.26387320679785603 - 3.1415926535897931) * 57.295779513082323, -0.84008767099694759 * 57.295779513082323},
+        // TODO: Check Karney
```

Comment:
again on my side
```
--------------karney
20003931.407432225,-1.3045327945402645e-05,-179.99998695467218,30010.522036068436,-0.99475954491334551
--------------vincenty
20003669.536581852,-7.591256469067261,-172.40874359800267,29487.135838230821,-0.99480541228839525
--------------andoyer
20018879.436128609,0,180,29935.015234617713,-0.99477272997811228
--------------thomas
0,0,0,0,1
--------------geographiclib 1.50.1
20003931.407432225,-1.3045327945402774e-05,-179.99998695467218,30010.522036068141,-0.99475954491334551
```

Are you sure you are getting that `distance` with `vincenty` ? (it is the same as `andoyer`)

Reply:
> Are you sure you are getting that distance with vincenty ?

My mistake, it should be ok now.

Reply:
> Are you sure you are getting that distance with vincenty ?

My mistake it should be ok now.